src/sys/bout_types.cxx

```diff
@@ -1,13 +1,22 @@
+#include <bout/assert.hxx>
```

Comment:
Should this now be `#include <boutexception.hxx>`?

Reply:
Yes, updated, thanks